### DISCUSSION TITLE
Add icon property to ToastMessageOptions

### DISCUSSION THREAD

**Discussion by jsodeman** [primefaces/community#1050] [PrimeVue]
The icon slot was added to the Toast component, but if you are triggering toast via the ToastService then there is no way to pass in an icon.

I suggest adding an option string property named "icon" to ToastMessageOptions for example:

`this.$toast.add({ severity: "success", summary: "Success", detail: "A message", life: 3000, icon: "fa-solid fa-circle-check" });`

**Comment by tugcekucukoglu** [comment]
We can wait for the community's demand.

Thank you!
